EIPS/eip-eth_simulateTransaction.md

```diff
@@ -0,0 +1,109 @@
+---
+eip: <to be assigned>
+title: eth_simulateTransaction - transaction dry runs
+author: Peter Jihoon Kim (@petejkim)
+discussions-to: <tbd>
```

Comment:
This field is required for a draft EIPs.  Recommend either a GitHub issue here (not this pull request) or a thread on https://ethereum-magicians.org/.